package.json

```diff
@@ -163,9 +165,6 @@
     "url": "^0.11.1",
     "webpack-bundle-analyzer": "^4.9.0"
   },
-  "resolutions": {
-    "**/gl": "^8.0.2"
-  },
```

Comment:
We still need this, right? Just that for Yarn 4, we need to change the format and remove the `**/`:

```
"resolutions": {
  "gl": "^8.0.2"
},
```

Reply:
i didn't touch this, it seems yarn removed that automatically (probably because gl isn't a dependency)

Reply:
> i didn't touch this, it seems yarn removed that automatically (probably because gl isn't a dependency), but i'll check again

Yes it's because yarn 4 doesn't understand the `**/gl` format so it think it's not a dependency. You need to manually change it to `gl` so that Yarn will pick it up as a transitive dependecy

Reply:
The lockfile isn't the issue, it's package.json

Reply:
hm, can we try this first? if we actually need 8.0.2 then it should be added as a dependency instead of forcing a resolution like this